src/components/flat-table/flat-table-row/flat-table-row.style.js

```diff
@@ -55,6 +60,16 @@ const StyledFlatTableRow = styled.tr`
   table-layout: fixed;
   width: auto;
 
+  ${StyledFlatTableCell} {
```

Comment:
I think you'll need to target `FlatTableCheckbox` and `FlatTableRowHeader` here as well

![image](https://user-images.githubusercontent.com/44157880/119501046-6392b080-bd60-11eb-984f-bd18a191f047.png)